cime_config/allactive/config_pesall.xml

```diff
@@ -2379,7 +2379,7 @@
       </pes>
     </mach>
   </grid>
-  <grid name="a%ne30.+oi%oARRM60to10">
+  <grid name="a%ne30.+oi%oARRM60to10|a%ne30.+oi%ARRM10to60">
```

Comment:
isn't the new grid name 'ARRM10to60E2r1'?

Reply:
It is, but you don't have to match the whole string.  This will match any other versions of the 10to60 grid "E2r2", etc, so provides some extensibility if it ever arises.  If it doesn't, it's no loss.  If we "E2r2" needs a different pe layout, then we add it then.

Reply:
great. Thanks for clarifying.